airbyte-integrations/connectors/source-paypal-transaction/source_paypal_transaction/manifest.yaml

```diff
@@ -150,11 +161,137 @@ definitions:
+  # TODO: Review if this is only a sandbox problem
+  search_invoices_stream:
+    type: DeclarativeStream
+    primary_key: id
+    name: "search_invoices"
+    retriever:
+      type: SimpleRetriever
+      record_selector:
+        $ref: "#/definitions/selector"
+      paginator:
+        type: DefaultPaginator
+        page_token_option:
+          type: RequestOption
+          inject_into: request_parameter
+          field_name: page
+        page_size_option:
+          inject_into: request_parameter
+          field_name: page_size
+          type: RequestOption
+        pagination_strategy:
+          type: PageIncrement
+          start_from_page: 1
+          page_size: 20
+      requester:
+        $ref: "#/definitions/requester"
+        http_method: POST
+        request_headers:
+          Content-Type: application/json
+        request_body_json:
+          creation_date_range:
+            start: "{{ config['start_date'] }}"
+            end: "{{ now_utc().strftime('%Y-%m-%dT%H:%M:%SZ') }}"    
+    $parameters:
+      field_path: items
+      path: "v2/invoicing/search-invoices"
+  
+  #New Stream
+  products_stream:
+    type: DeclarativeStream
+    primary_key: id
+    name: "products"
+    retriever:
+      type: SimpleRetriever
+      record_selector:
+        $ref: "#/definitions/selector"
+      paginator:
+        type: DefaultPaginator
+        pagination_strategy:
+          type: PageIncrement
+          start_from_page: 1
+          page_size: 10
+        page_token_option:
+           type: RequestOption
+           inject_into: request_parameter
+           field_name: page
+        page_size_option:
+           inject_into: request_parameter
+           field_name: page_size
+           type: RequestOption
+      requester:
+        $ref: "#/definitions/requester"
+        http_method: GET
+        request_headers:
+          Content-Type: application/json
+    $parameters:
+      path: "v1/catalogs/products"
+      field_path: products
+
+  #Stream List Disputes
+  list_disputes_stream:
+    type: DeclarativeStream
+    primary_key: dispute_id
+    name: "list_disputes"
+    retriever:
+      type: SimpleRetriever
+      record_selector:
+        $ref: "#/definitions/selector"
+      paginator:
+        type: DefaultPaginator
+        page_token_option:
+          type: RequestOption
+          inject_into: request_parameter
+          field_name: next_page_token
+        page_size_option:
+          inject_into: request_parameter
+          field_name: page_size
+          type: RequestOption
+        pagination_strategy:
+            type: PageIncrement
+            start_from_page: 1
+            page_size: 50
+      requester:
+        $ref: "#/definitions/requester"
+        http_method: GET
+        request_parameters:
+          start_time: "{{ stream_slice.get('update_time_after', config['start_date']) }}"   
+    # incremental_sync:
```

Comment:
What's going on with incremental?

Reply:
I was testing it, and I forgot to remove it from the commit.